packages/components/src/services/ForumService.ts

```diff
@@ -0,0 +1,55 @@
+import { HttpService } from '../utils/httpService';
+import { CreateQuestion, ForumPosts } from '../schemas/Forum.schema';
+
+const apiUrl = 'https://breakup-app-api.herokuapp.com/api';
```

Comment:
can we keep this URL in some other file, put it in HttpService.ts

Reply:
will be added to env variable in further PR's